doc/classes/AudioStream.xml

```diff
@@ -28,6 +28,12 @@
 			<description>
 			</description>
 		</method>
+		<method name="_get_parameter_list" qualifiers="virtual const">
+			<return type="Array" />
+			<description>
+				Return the controllable parameters of this stream. This array must be interleaved as a dictionary with a property info description (see [method Object.get_property_list]) and the default value for it.
```

Comment:
I think I don't fully get this description. Is the expected format `[ Dictionary, default, Dictionary, default]`? Why the default can't be part of the Dictionary? PropertyInfo would just ignore it.

Reply:
Thats a good point, will change to that.